### PR TITLE
Makes AI Less Cheeseable

### PR DESCRIPTION
**Pull request by JamieD1**
# Github documenting your Pull Request

Makes AI Less Cheeseable

image

# Wiki Documentation

AI Image probably will need updating

# Changelog

Edit the changelog below to reflect the changes made by this PR, even if the changes are minor - required for every PR.
If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name.

:cl:  
tweak: Makes AI Less Cheeseable 
/:cl:

### CODE REVIEW

**Comment by Bibby0110** [comment]
The pain of having a silicon main head dev.

This is just massive power creep, the biggest problem is solved by a regular windoor, let alone extra turrets and a reinforced one

**Comment by JamieD1** [comment]
> The pain of having a silicon main head dev.
> 
> This is just massive power creep, the biggest problem is solved by a regular windoor, let alone extra turrets and a reinforced one

Fuck off bibby, its way too easy as of now. The only obstacle is the r walls.

**Comment by Jarod1200** [comment]
I like the new shape and turrets but I think the apc shouldn't be inside the central ai cube

**Comment by cuackles** [comment]
Needs more walls

**Comment by Hopekz** [comment]
Common man, you're better than this. You're supposed to write down what changes you made.

"Making the AI less cheesable" isn't enough.
PR's have technical language on what changed, like "added an extra layer of reinforced walls to x." or mention it is a redesign, anything about your changes.

Stop making the standards for PR posting lower since you're supposed to be an example of what to do.

**Comment by JamieD1** [comment]
> Common man, you're better than this. You're supposed to write down what changes you made.
> 
> "Making the AI less cheesable" isn't enough.
> PR's have technical language on what changed, like "added an extra layer of reinforced walls to x." or mention it is a redesign, anything about your changes.
> 
> Stop making the standards for PR posting lower since you're supposed to be an example of what to do.

Look at the picture, picture is worth a thousand words.

**Comment by adamsong** [comment]
image

**Comment by Cartlord** [comment]
I genuinely do not see how this is necessary, AI never seemed "cheesable" in a way that this actually fixes

**Comment by adamsong** [comment]
This prevents using a launchpad to card the AI

**Comment by Marmio64** [comment]
only thing necessary are the turrets in front of the windoors honestly, everything else is too much and is overtly buffing the AI

**Comment by Moggram** [comment]
> 
> 
> This prevents using a launchpad to card the AI

Can't you just teleport on top of it and not get hit?

**Comment by Partheo** [comment]
Yeah the PR body tells me nothing about what this changes, bad changelog too

**Comment by Darthkill** [comment]
> This prevents using a launchpad to card the AI

You can drop in that convenient hidey hole, or in the AI's face and drop into the hidey hole to shelter. I dont see what cheese this is defending against.

**Comment by maxion12345** [comment]
> I genuinely do not see how this is necessary, AI never seemed "cheesable" in a way that this actually fixes

basically as it is now, with the APC on the bottom, you can remove two rwalls and then just break the apc with a rock or something

it used to be more "fight through a room with turrets and walls to get to the core, then breach the core to destroy the AI" but now all you need is some form of space protection and a toolbox
you can probably just run up to the APC with the tools and snip wires if you are fast enough so maybe you won't even need a gun

tl;dr
current AI core is extremely cheeseable thanks to the APC being exposed on the bottom

**Comment by Hopekz** [comment]
> it used to be more "fight through a room with turrets and walls to get to the core, then breach the core to destroy the AI" but now all you need is some form of space protection and a toolbox

We can always move the AI core back into the center of the station again as God intended.

For some reason, tcomms vs the AI out of all things received the same high security that being in the middle of the station provides.
